parent/pom.xml

```diff
@@ -1094,6 +1102,18 @@
         <version>${version.identity}</version>
       </dependency>
 
+      <!--      JMH Tests   -->
```

Comment:
❓   how come this moved?

Reply:
Same as below, just tidying things up and moving it above the dependency convergence section.